src/buildVariables.ts

```diff
@@ -191,17 +256,26 @@ const buildUpdateVariables = (introspectionResults: IntrospectionResult) => (
 ) => {
   return Object.keys(params.data).reduce(
     (acc, key) => {
-      if (Array.isArray(params.data[key])) {
-        const inputType = findInputFieldForType(
-          introspectionResults,
-          `${resource.type.name}UpdateInput`,
-          key
-        );
+      // Put id field in a where object
+      if (key === 'id' && params.data[key]) {
+        return {
+          ...acc,
+          where: {
+            id: params.data[key]
+          }
+        };
+      }
+      const inputType = findInputFieldForType(
+        introspectionResults,
+        `${resource.type.name}UpdateInput`,
+        key
+      );
 
-        if (!inputType) {
-          return acc;
-        }
+      if (!inputType) {
+        return acc;
+      }
```

Comment:
i am not sure if there are some cases where we should add the property despite there is no input type.

when key is `id` is one exception, that's why i moved it on top of the function. But maybe there are other cases?